client/state/data-layer/wpcom-http/pipeline/remove-duplicate-gets/index.js

```diff
@@ -41,6 +41,18 @@ export const clearQueue = () => {
  */
 const isGetRequest = ( request ) => 'GET' === get( request, 'method', '' ).toUpperCase();
 
+/**
+ * Returns all elements that exist in any of the two arrays at least once,
+ *
+ * @param {Array} a First array
+ * @param {Array} b Second array
+ * @returns {Array} Array of elements that exist in at least one of the arrays
+ */
+const unionWith = ( a = [], b = [] ) =>
```

Comment:
Default values are necessary to ensure that we're not trying to spread an `undefined`. Without the default values, tests are going to fail.